### DISCUSSION TITLE
Question on usage

### DISCUSSION THREAD

**Discussion by blaidd31204** [danielo515/obsidian-modal-form#355] [Q&A]
I'd like to take some pdfs like the world builder's Guide book and make form templates in Obaidian for documenting and building NPCs, locations, adventures, etc. that I would have in my vault as notes.  Will this help me do something like that?

**Comment by danielo515** [comment]
Well, that is not a very extensive description. It depends on how having a
form will benefit you to input data. Currently uploading files via the form
is not yet supported though

**Comment by blaidd31204** [comment]
What I wanted to do was use the PDF's as a foundation. In Adobe PDF, you can take a pdf and create a form from te scan. I wanted to have something that I could use to generate Obsidian templates from a pdf form to use them in my world building.  Does that provide more clarity?

**Comment by danielo515** [comment]
I am not familiar with thar way of working. Why do you need the PDF as base
? You can define a form using the plugin UI that mimics your pdf. Other
than that, it doesn't fit the workflow you described
